hub/api/dataset_utils.py

```diff
@@ -0,0 +1,39 @@
+import numcodecs
+from codecs import PngCodec, Lz4, NumPy, Zstd, JpegCodec
+
+
+def _get_compressor(compressor: str, **kwargs):
```

Comment:
instead of doing a function here, i recommend using a structure similar to what [i've been using during development](https://github.com/activeloopai/Hub/blob/32f2dc3367955d7b7959275792b770c824d098dc/hub/core/chunk_engine/dummy_util.py#L38)

if you support `**kwargs` for these inputs, then you can just get the compressor class from the map & do `compressor_class(**kwargs)`.